Automattic/wp-calypso
# Add Mailchimp feature as a marketing tool

Related to #91603

## Proposed Changes

* It adds Mailchimp feature as a WPCOM marketing tool.

## Testing Instructions



* In the left menu, navigate to Tools > Marketing.
* Check that you see the new Mailchimp feature.

<img width="479" alt="Screenshot 2024-06-11 at 16 34 15" src="https://github.com/Automattic/wp-calypso/assets/876340/19e498ce-dc5f-4007-bd77-c920682b9d4b">

## Pre-merge Checklist
